ui/app/components/ui/popover/popover.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Popover from './popover.component'
+import { text } from '@storybook/addon-knobs/react'
+
+const containerStyle = {
+  width: 800,
+  height: 600,
+  background: 'pink',
+  position: 'relative',
+}
+
+const preventDefault = (event) => {
+  event.preventDefault()
```

Comment:
Is this still needed now that a `div` isn't being used for the backdrop? I had hoped it wouldn't be.

Maybe a storybook `action` could be passed in instead, to show that the clicking works.